python/adelphi/bin/test-adelphi

```diff
@@ -0,0 +1,86 @@
+#!python
+
+# Run test suite for some set of Cassandra versions.
+#
+# We implement this as a front-end script because the tox/pytest/unittest chain
+# isn't great about iterating over a suite of test fixtures and re-running the
+# collected tests for each of them.  Rather than fight with the frameworks we
+# manage the fixture iteration manually in this script.  This also has the
+# nice side effect of moving a lot of C* checking/session code out of the test
+# suite, which in turn should allow us to write simpler tests.
+import configparser
+import os
+
+from tests.util.cassandra_util import connectToLocalCassandra
+
+import click
+import docker
+from tenacity import retry, stop_after_attempt, wait_fixed
+import tox
+
+
+# Default C* versions to include in all integration tests
+DEFAULT_CASSANDRA_VERSIONS = ["2.1.22", "2.2.19", "3.0.23", "3.11.9", "4.0-rc1"]
```

Comment:
One thing I noticed is that we don't have 3.11.10 in the list, probably makes sense to include that since it's the latest stable 3.11 release.

I'll create a separate issue to add that to the mix.

Reply:
-> #157

Reply:
@jdonenine This was easy enough to address now so I just did it... https://github.com/datastax/adelphi/pull/152/commits/98fa005cef22aa752596672a21af6f4622e7b486